lib/msf_autoload.rb

```diff
@@ -0,0 +1,167 @@
+require "zeitwerk"
+
```

Comment:
If we're treating this file as all of the things we need to fix with namespacing at some point; What are your thoughts on keeping track of the other files we want to fix later as comments? i.e. namespacing wmap/events properly. At the very least it won't be lost in a ticket